src/docfx/build/legacy/files/LegacyTableOfContents.cs

```diff
@@ -74,6 +67,9 @@ private sealed class LegacyTableOfContentsMetadata
 
             [JsonProperty(PropertyName = "pdf_name")]
             public string PdfName { get; set; }
+
+            [JsonExtensionData]
+            public JObject Metadata { get; set; }
```

Comment:
Is `experimental` and `experimental_id` needed for non-legacy output ?

Reply:
yes, it's used for AB testing

Reply:
and we just pass through that, don't need to parse the metadata, it's wrote by writer